framework/src/dirackernels/VectorPostprocessorPointSource.C

```diff
@@ -20,28 +20,46 @@ VectorPostprocessorPointSource::validParams()
 {
   InputParameters params = DiracKernel::validParams();
 
-  params.addClassDescription("Apply a point load defined by VectorPostprocessor.");
+  params.addClassDescription("Apply a point load defined by Reporter or VectorPostprocessor.");
 
-  params.addParam<VectorPostprocessorName>(
+  params.addParam<std::string>(
       "vector_postprocessor",
-      "The name of the VectorPostprocessor containing positions and corresponding load values");
-  params.addParam<std::string>("x_coord_name", "x", "name of column containing x coordinates.");
-  params.addParam<std::string>("y_coord_name", "y", "name of column containing y coordinates.");
-  params.addParam<std::string>("z_coord_name", "z", "name of column containing z coordinates.");
-  params.addParam<std::string>("value_name", "value", "name of column containing values.");
+      "The name of the VectorPostprocessor containing positions and corresponding load values.  "
+      "This will be appended onto the reporter name as <vector_postprocessor>/<vector_name>.");
+  params.addParam<std::string>("x_coord_name", "x", "name of reporter containing x coordinates.");
+  params.addParam<std::string>("y_coord_name", "y", "name of reporter containing y coordinates.");
+  params.addParam<std::string>("z_coord_name", "z", "name of reporter containing z coordinates.");
+  params.addParam<std::string>("value_name", "value", "name of reporter containing values.");
```

Comment:
```suggestion
  params.addParam<ReporterName>("x_coord_name", "x", "name of reporter containing x coordinates.");
  params.addParam<ReporterName>("y_coord_name", "y", "name of reporter containing y coordinates.");
  params.addParam<ReporterName>("z_coord_name", "z", "name of reporter containing z coordinates.");
  params.addParam<ReporterName>("value_name", "value", "name of reporter containing values.");
```

Reply:
Huh, I'm confused. The parameters don't make any sense anymore. The VPP was supposed to have coords and data, and teh next params were just the column names in that VPP, now they are separate reporters?

Reply:
Reporter names have the syntax `object_name/value_name`. So the parameter `vector_postprocessor` will specify the `object_name` and the `x_coord_name`, `y_coord_name`, etc. specify the `value_name`.

Reply:
Ok, then the doc strings need to be fixed

Reply:
The api doesn't change so it will still work as it did before.  The reporter interface makes is written like this so that a consumer can take in different reporters.  It's not helpful in this case but there are examples of using different reporters in the tests.  I think I want to pass around a reporter in the optimization code and I need this vppPointSource to apply the misfit so it would be nice if it could read from a reporter.

Reply:
I don't know what you mean by this

> Ok, then the doc strings need to be fixed

Reply:
`x_coord_name` is not "name of reporter", it is the name of the value

Reply:
@dschwen or @zachmprince I fixed the doc strings and it passes the tests.  Will you merge it?

Reply:
@dschwen will you review this so that it can be merged

Reply:
@dschwen will you merge this?